app/helpers/pdf/work_days.py

```diff
@@ -120,6 +122,17 @@ def _get_detail_columns(include_support=False, include_expenditures=False):
         ),
     ]
 
+    if include_transfers:
+        columns.append(
+            Column(
+                name="transfer",
```

Comment:
Sometimes column name is `ActivityType.TRANSFER.value` and sometimes `"transfer"` directly. Can it cause any issue if updating one but not the other?

Reply:
I did a bit of cleaning to leverage more on ActivityTypes.xxx.value and same for expenditures